PIPs/pip-0003.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Removing sequence field from transactions
+author: Mostafa Sedaghat Joo (@b00f)
+status: Draft
+type: Standard
+created: 2023-08-23
+---
+
+## Abstract
+
+This document propose to remove the `Sequence` field from the transactions.
+
+## Motivation
+
+The `Sequence` in Pactus is similar to the `nonce` in Ethereum.
```

Comment:
Please add transaction data in code here for clean Motivation details.

```go
type Tx struct {
	memorizedID  *ID
	basicChecked bool

	data txData
}

type txData struct {
	// Version format
	//  7             0
	// +-+-+-+-+-+-+-+-+
	// |L|R|R|R|VERSION|
	// +-+-+-+-+-+-+-+-+
	// L: Lock Time transaction
	// R: Reserved bit
	//
	Version   uint8
	Stamp     hash.Stamp
	LockTime  uint32
	Sequence  int32
	Fee       int64
	Payload   payload.Payload
	Memo      string
	PublicKey crypto.PublicKey
	Signature crypto.Signature
}
```

Reply:
We can add link to the website: https://pactus.org/learn/transaction/format/

Feel free to edit it